plugins/thehive/Dockerfile

```diff
@@ -15,6 +19,7 @@ ADD . /python/src
 RUN	python setup.py build && python setup.py install
 
 # User to run plugin code. The two supported users are: root, nobody
-USER nobody
+# Needed for ssl
+USER root
```

Comment:
Is it really necessary?

Reply:
I copied it from another plugin that I saw also contained the same comment, and since it was SSL issues I thought it couldn't hurt to include

Reply:
Can we test if this works with nobody? I know we have other plugins where SSL cert file is required